layouts/index.html

```diff
@@ -181,7 +179,7 @@ <h5>Curious where it all began?</h5>
       <section id="uses" class="content-center">
         <div class="content-center mb2">
           <h2>IPFS can help here and now</h2>
-          <h5>No matter what you do with the web, IPFS is useful today.</h5>
+          <h5>No matter what you do with the web, IPFS can help make it better today.</h5>
```

Comment:
I prefer to keep the active verb instead of `can help`, which is passive voice.

Reply:
"No matter what you do with the web, IPFS helps make it better today."